docs/projects/create-project.md

```diff
@@ -254,4 +256,5 @@ Example:
 
 ## Deleting a project
 
-See {py:class}`~mlrun.db.httpdb.HTTPRunDB.delete_project`.
\ No newline at end of file
+See {py:class}`~mlrun.db.httpdb.HTTPRunDB.delete_project`. Deleting a project also deletes the project's objects (artifact metadata, runs, etc.). 
```

Comment:
When we delete a project, we do not delete the artifact data

Reply:
not the files, as written in the next sentence. But the metadata is deleted. Correct?

Reply:
The metadata is only removed when the project is deleted with the `deletion_strategy=cascade` option, which is not the default value
https://docs.mlrun.org/en/stable/api/mlrun.db.html#mlrun.db.httpdb.HTTPRunDB.delete_project

Reply:
true, but if there are resources, you can't use the default option. Therefore you can't delete a project and leave the metadata.